lmdeploy/tokenizer.py

```diff
@@ -181,10 +185,11 @@ def __call__(self, s: Union[str, Sequence[str]]):
             list[int]: token ids
         """
         import addict
-        add_bos = False
-        add_eos = False
 
-        input_ids = self.model.Encode(s, add_bos=add_bos, add_eos=add_eos)
+        input_ids = self.model.Encode(s,
+                                      add_bos=add_bos,
+                                      add_eos=add_eos,
+                                      **kwargs)
```

Comment:
What's the difference from previous version?

Reply:
previous version does not passing kwargs in.